lms/static/sass/course/layout/_courseware_preview.scss

```diff
@@ -158,6 +158,10 @@
       color: theme-color("inverse");
     }
 
+    .toggle-show-more {
```

Comment:
I'm not seeing this CSS applied on my devstack. I did compile the sass. I thought that button on the course outline banner looked a little off and was wondering whether it was related to this. I'll look into it tomorrow.

Reply:
You're right; this rule was in the wrong place. I've removed it and the corresponding class since it looked how I wanted it to without it.